benchmarks/benchmarks/kernel/bench_gsddmm_u_dot_v.py

```diff
@@ -0,0 +1,33 @@
+import time
+import dgl
+import torch
+
+from .. import utils
+
+def calc_gflops(graph, feat_size, num_heads, time):
+    return round(2 * graph.num_edges() * feat_size / 1000000000 / time, 2)  # count both mul and add
+
+@utils.benchmark('flops', timeout=600)
+@utils.parametrize('graph', ['ogbn-arxiv', 'reddit', 'ogbn-proteins'])
+@utils.parametrize('feat_size', [4, 32, 256])
```

Comment:
We should clarify the definition of `feat_size` (especially when number of heads is not 1).

Reply:
added description